### PR TITLE
Docker file additions / CSS changes

### PR DESCRIPTION
**Pull request by jxb345**
- Adjusted stying in components
- Added docker-related files
- included requested `app.use` line to make bundle.js file accessible

### CODE REVIEW

File: client/src/components/AboutAuthorHeader.jsx [hunk 1]
```diff
@@ -7,16 +7,29 @@ const Paragraph = styled.p`
   height: 24px;
   width: auto;
   font-weight: bold;
-  font-size: 12px;
+  font-size: 15px;
```
Comment (MuhammadJamaluddin) [inline client/src/components/AboutAuthorHeader.jsx:10]: https://github.com/rpt12-knightrider/mj-service/pull/8#discussion_r273290233

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -21,6 +21,7 @@ class App extends React.Component {
 
   getInfo(id) {
     $.ajax({
+      // change 'localhost' for deployment
```
Comment (MuhammadJamaluddin) [inline client/src/components/App.jsx:24]: Exactly

File: client/src/components/AuthorName.jsx [hunk 3]
```diff
@@ -2,25 +2,30 @@ import React from 'react';
 import FollowButton from './FollowButton.jsx';
 import styled from 'styled-components';
 
+const Div = styled.div`
+  display: flex;
+  /* font-size:0; */
```
Comment (MuhammadJamaluddin) [inline client/src/components/AuthorName.jsx:7]: I would remove all the commented/unused styles. & btw, you would definitely need a clean up day for the dead & non-conforming style guide code 😄. 
Also be sure to restructure your project files & update the `README `

File: client/src/components/AuthorName.jsx [hunk 4]
```diff
@@ -2,25 +2,30 @@ import React from 'react';
...
+`;
+
 const Name = styled.p`
   font-family: "Merriweather", "Georgia", serif;
-  font-size: 16px;
+  font-size: 18px;
```
Comment (MuhammadJamaluddin) [inline client/src/components/AuthorName.jsx:12]: same as before

File: client/src/components/AuthorName.jsx [hunk 5]
```diff
@@ -40,13 +45,17 @@ const Button = styled.button`
 class AuthorName extends React.Component {
   render() {
     return (
-      <div>
+
+      // consider: https://developer.mozilla.org/en-US/docs/Web/CSS/clear
+      <Div>
```
Comment (MuhammadJamaluddin) [inline client/src/components/AuthorName.jsx:50]: Did you consider a better naming?... `Wrapper` for instance.

File: client/src/components/BooksBy.jsx [hunk 6]
```diff
@@ -7,16 +7,29 @@ const Name = styled.p`
...
+    padding: 0;
+    width: 400px;
 `;
 
 class BooksBy extends React.Component {
   render() {
     return (
+      <div>
       <Name>Books By {this.props.name}</Name>
+      <HR></HR>
```
Comment (MuhammadJamaluddin) [inline client/src/components/BooksBy.jsx:31]: A room for a more clear name.

File: client/src/components/AuthorName.jsx (hunk 3 above)
Comment (jxb345) [reply client/src/components/AuthorName.jsx:7]: Yes, cleaning up my code is on my list, though it's not a priority right now. Also, re: restructure your project files - for this point, do you have something specific in mind?

Comment (MuhammadJamaluddin) [reply client/src/components/AuthorName.jsx:7]: Essentially using the `server` directory to include the back-end related code.

**Pull request merged**